### ISSUE TITLE
Missing translations

### ISSUE THREAD

**Issue by eson57** [poetaster/harbour-beamdimension#2]
The Jolla Store version doesn't contain any translation files.

**Comment by poetaster** [comment]
Yes, no translations have been done to date. The German translation I can do as I have time. Currently, I'm busy with bug squashing on other projects. If you can contribute a translation I'd be happy to include it!

**Comment by poetaster** [comment]
Ah, I'm wrong there is a swedish ranslation. I'm checking.

**Comment by eson57** [comment]
Yes, I did Swedish translation for both BeanCalc and ConcreteMixer, as I am a construction engineer.

**Comment by poetaster** [comment]
It is comforting to know that there is a Swedish civil engineer on board! 

I've uploaded newly built packages. Also to openrepos. I believe that the issue arose in using the 3.6.x sdk, but I'm not sure. It's a bit odd since I had only rebuilt to insure an aarch version was showing up. Which is silly since these are pure QML apps. In any case, they seem to need a little love.

Let me know if you see any suspect math!

Thanks,

Mark
